internal/pkg/project/manifest/manifest_test.go

```diff
@@ -308,219 +312,193 @@ func fullJson() string {
 `
 }
 
-func fullStruct() *Content {
-	return &Content{
-		Version: 2,
-		Project: Project{
-			Id:      12345,
-			ApiHost: "foo.bar",
+func fullRecords() []model.ObjectManifest {
+	return []model.ObjectManifest{
+		&model.BranchManifest{
+			RecordState: model.RecordState{
+				Persisted: true,
+			},
+			BranchKey: model.BranchKey{
+				Id: 10,
+			},
+			Paths: model.Paths{
+				AbsPath: model.NewAbsPath(
+					"",
+					"main",
+				),
+			},
 		},
-		SortBy: model.SortById,
-		Naming: naming.TemplateWithIds(),
-		Filter: model.Filter{
-			AllowedBranches:   model.AllowedBranches{"foo", "bar"},
-			IgnoredComponents: model.ComponentIds{"abc"},
+		&model.BranchManifest{
+			RecordState: model.RecordState{
+				Persisted: true,
+			},
+			BranchKey: model.BranchKey{
+				Id: 11,
+			},
+			Paths: model.Paths{
+				AbsPath: model.NewAbsPath(
+					"",
+					"11-dev",
+				),
+			},
 		},
-		Branches: []*model.BranchManifest{
```

Comment:
Predtym sa validovat obsah nacitaneho suboru, samostatne boli `Branches`, a `Configs (ConfigManifestWithRows)`.
Teraz je to jedno pole zaznamov.

Je to tak lepsie, lebo nevalidujeme iba nacitanie suboru, ale vysledny interny stav.